addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.xml

```diff
@@ -145,7 +145,6 @@
                     </t>
                 </t>
             </t>
-            <div class="d-none d-print-block text-wrap" t-out="label"/>
```

Comment:
Removing this line will entirely hide the info in print mode 😉

Reply:
The print view is less important. 
At this stage, If we can not find a solution that is working for print without (negative) impact on the "normal" we should revert it but of course the best is to find a solution for both

Reply:
I'm lookin for a potential t-if that could allow the div to stay but I'm not that experimented with regards to the print view's specificities in a browser so i have to dig to find something. I'll let you know.

Reply:
LTM